lib/main.dart

```diff
@@ -74,7 +74,7 @@ class _MyAppState extends State<MyApp> {
           onPressed: () {
             _smkitUiFlutterPlugin.startAssessment(
               type: AssessmentTypes.custom,
-              assessmentID: "YOUR_ASSESSMENT_ID, // If you dont have a assessmentID please use null
+              assessmentID: "YOUR_ASSESSMENT_ID", // If you dont have a assessmentID please use null
```

Comment:
@SencyNetanel why not making this argument optional?

also, don't we need to add this change in README.md?